pycqed/measurement/quantum_experiment.py

```diff
@@ -661,5 +662,81 @@ def save_timers(self, quantum_experiment=True, sequence=True, segments=True, fil
             raise e
 
 
+    def plot(self, sequences=None, segments=None, qubits=None,
+             save=False, **plot_kwargs):
+        """
+        Plots (a subset of) sequences / segments of the QuantumExperiment
+        :param sequences (int, list): sequences to plot. Can be None (plot all
+        sequences), an integer (index of sequence to plot),  or a list of
+        integers/str. If strings are in the list, then plots only sequences
+        with the corresponding name.
+        :param segments (int, list): Segments to be plotted.
+            If a single index i is provided, then the ith segment will be plot-
+            ted for each sequence in `sequences`. Otherwise a list of list of
+            indices must be provided: the outer list corresponds to each
+            sequence and the inner list to the indices of the segments to plot.
+            E.g. segments=[[0,1],[3]] will plot the 0th and 1rst segment of
+                sequence 0 and the 3d segment of sequence 1.
+            Plots all segments by default.
+        :param qubits (list): list of qubits to plot. Defaults to self.meas_objs
```

Comment:
comment still open:
- please clarify whether list of objects or list of strings (names). Consider passing it to self.get_qubits to allow the user passing qb names.